src/twisted/internet/_sslverify.py

```diff
@@ -21,6 +22,109 @@
 from twisted.python import log
 from twisted.python._oldstyle import _oldStyle
 
+# If we don't have TLS v1.3 yet, we can't disable it -- this is just so when it
+# makes it into OpenSSL, connections end up going to v1.3
+if not hasattr(SSL, "OP_NO_TLSv1_3"):
```

Comment:
Don't ever use `hasattr`.  It masks exceptions.  Use `getattr(o, n, default) is not default`.

In this case, you can just do `_OP_NO_TLSv1_3 = getattr(SSL, "OP_NO_TLSv1_3", 0x00)` though.